Emulator/pal/gc.asm

```diff
@@ -10,15 +10,13 @@ TCHR,	0000					/TEMP CHAR
 
 		*200					/LOCATE @ 0200
 
-		CLA						/CLEAR ACC
-		CLL						/CLEAR LINK
+		CLA CLL					/CLEAR ACC, CLEAR LINK
```

Comment:
palbart will combine microcoded instructions if you write them on the same line.